eventio/__init__.py

```diff
@@ -6,7 +6,7 @@
 from .histograms import Histograms
 
 
-__version__ = '1.5.1.post1'
+__version__ = '1.5.1.post2'
```

Comment:
I guess this means that there will be a new release after the merging of this PR?

Reply:
Yes. Eventio does not yet use setuptools scm.